telegram/chat.py

```diff
@@ -53,11 +53,11 @@ class Chat(TelegramObject):
     def __init__(self,
                  id,
                  type,
-                 title='',
-                 username='',
-                 first_name='',
-                 last_name='',
-                 all_members_are_admins=False,
+                 title=None,
+                 username=None,
+                 first_name=None,
+                 last_name=None,
+                 all_members_are_admins=None,
```

Comment:
The API of Chat calls this var as `all_members_are_administrators`, here we use `admins`. Is there anything that translates `admins`. I think we actually have a bug here, or telegram might have silently changed the API.